src/wasm-lib/src/wasm.rs

```diff
@@ -614,3 +614,18 @@ pub fn calculate_circle_from_3_points(ax: f64, ay: f64, bx: f64, by: f64, cx: f6
         radius: result.radius,
     }
 }
+
+/// Takes a kcl string and Meta settings and changes the meta settings in the kcl string.
+#[wasm_bindgen]
+pub async fn change_kcl_units(code: &str, settings_str: &str) -> Result<String, String> {
```

Comment:
so @franknoirot you pass me the kcl code, then i give you kcl code back, then yeah it will trigger a re-parse but thats fine

Reply:
Great, that works